runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -28,11 +34,94 @@
 /**
  * Created by mwei on 12/9/15.
  */
-@Slf4j
 @Accessors(chain = true)
 public class CorfuRuntime {
 
-    @Data
+    private static final Logger log = org.slf4j.LoggerFactory.getLogger(CorfuRuntime.class);
+
+    public static String getMpASV() {
+        return CorfuRuntime.mpASV;
+    }
+
+    public static String getMpLUC() {
+        return CorfuRuntime.mpLUC;
+    }
+
+    public static String getMpCR() {
+        return CorfuRuntime.mpCR;
+    }
+
+    public static String getMpObj() {
+        return CorfuRuntime.mpObj;
+    }
+
+    public static MetricRegistry getMetrics() {
+        return CorfuRuntime.metrics;
+    }
+
+    public CorfuRuntimeParameters getParameters() {
+        return this.parameters;
+    }
+
+    public LayoutView getLayoutView() {
```

Comment:
Annotate with Lombok @Getter all fields you return with this and remove these methods.  - CorfuRuntime is big enough